rule_unspecified.go

```diff
@@ -0,0 +1,15 @@
+// +build !linux
+
+package netlink
+
+func (h *Handle) RuleAdd(rule *Rule) error {
```

Comment:
Same here. Unimplemented methods with pointer receiver `(Handle*)` must go in `handle_unspecified.go`

Reply:
done! moved them to handle_unspecified.go